platform/linux-generic/arch/aarch64/cpu_flags.c

```diff
@@ -23,6 +23,10 @@ typedef struct {
 	bool valid;
 } hwcap_feat_flag_t;
 
+/* Linux HWCAP and HWCAP2 flags
+ *
+ * See https://docs.kernel.org/arm64/elf_hwcaps.html for meaning of each flag.
```

Comment:
"/arch/" seems to be missing (https://docs.kernel.org/arch/arm64/elf_hwcaps.html).

Reply:
Fixed in v2.